tests/beaker_tests/cisco_portchannel_global/test_portchannel_global.rb

```diff
@@ -469,49 +509,91 @@ def test_harness_portchannel_global(tests, id)
-    tests['non_default_properties_no_rotate'][:manifest_props]['mac'] = 'port'
-    tests['non_default_properties_no_rotate'][:resource_props]['bundle_hash'] = 'port'
-    test_harness_portchannel_global(tests, id)
-
-    tests[id][:desc] = '2.5 Non Default Properties'
-    tests['non_default_properties_no_rotate'][:manifest_props]['port'] = 'port-only'
-    tests['non_default_properties_no_rotate'][:resource_props]['bundle_hash'] = 'port-only'
-    tests['non_default_properties_no_rotate'][:manifest_props]['dst'] = 'src-dst'
-    tests['non_default_properties_no_rotate'][:resource_props]['bundle_select'] = 'src-dst'
-    test_harness_portchannel_global(tests, id)
-
-    tests[id][:desc] = '2.6 Non Default Properties'
-    tests['non_default_properties_no_rotate'][:manifest_props]['port-only'] = 'ip-gre'
-    tests['non_default_properties_no_rotate'][:resource_props]['bundle_hash'] = 'ip-gre'
-    test_harness_portchannel_global(tests, id)
-
-    tests[id][:desc] = '2.7 Non Default Properties'
-    tests['non_default_properties_no_rotate'][:manifest_props]['src-dst'] = 'dst'
-    tests['non_default_properties_no_rotate'][:resource_props]['bundle_select'] = 'dst'
-    test_harness_portchannel_global(tests, id)
+    if tests[:resilient_unsupported]
+      id = 'non_default_properties_no_resilient'
+      tests[id][:desc] = '2.1 Non Default Properties'
+      test_harness_portchannel_global(tests, id)
+
+      tests[id][:desc] = '2.2 Non Default Properties'
+      tests['non_default_properties_no_resilient'][:manifest_props]['ip-only'] = 'ip-gre'
+      tests['non_default_properties_no_resilient'][:resource_props]['bundle_hash'] = 'ip-gre'
+      tests['non_default_properties_no_resilient'][:manifest_props]['src-dst'] = 'src'
+      tests['non_default_properties_no_resilient'][:resource_props]['bundle_select'] = 'src'
+      test_harness_portchannel_global(tests, id)
+
+      tests[id][:desc] = '2.3 Non Default Properties'
+      tests['non_default_properties_no_resilient'][:manifest_props]['ip-gre'] = 'mac'
+      tests['non_default_properties_no_resilient'][:resource_props]['bundle_hash'] = 'mac'
+      tests['non_default_properties_no_resilient'][:manifest_props]['src'] = 'dst'
+      tests['non_default_properties_no_resilient'][:resource_props]['bundle_select'] = 'dst'
+      test_harness_portchannel_global(tests, id)
+
+      tests[id][:desc] = '2.4 Non Default Properties'
+      tests['non_default_properties_no_resilient'][:manifest_props]['mac'] = 'port'
+      tests['non_default_properties_no_resilient'][:resource_props]['bundle_hash'] = 'port'
+      test_harness_portchannel_global(tests, id)
+
+      tests[id][:desc] = '2.5 Non Default Properties'
+      tests['non_default_properties_no_resilient'][:manifest_props]['port'] = 'port-only'
+      tests['non_default_properties_no_resilient'][:resource_props]['bundle_hash'] = 'port-only'
+      tests['non_default_properties_no_resilient'][:manifest_props]['dst'] = 'src-dst'
```

Comment:
Most if not all of this goes away with a rewrite, but it caught my eye. You don't have `port`, `src`, `dst`, `mac`, etc property names in your type, so what are these settings doing?